i18n/po/i18n_nl.po

```diff
@@ -202,6 +420,11 @@ msgstr ""
 msgid "Reference model schema(s) {0} specified in options not valid or not found in schema directories"
 msgstr ""
 
+#: ../tools/src/main/java/com/nedap/archie/archetypevalidator/validations/AuthoredArchetypeMetadataChecks.java:37
+#, java-format
+msgid "Resource description language {0} has an incorrect key: {1}"
+msgstr ""
```

Comment:
Translation missing

Reply:
This is a very technical sentence, that can only be found by editing the object model or introducing a bug - not by editing ADL. Not translating at the moment.